chainerrl/agents/double_iqn.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()  # NOQA
+
+import chainer
+import chainer.functions as F
+
+from chainerrl.agents import iqn
+
+
+class DoubleIQN(iqn.IQN):
+    """Double IQN. Using the primary Q-network's greedy/max action
+
+    to compute the target value rather than use the target network's
+    max action.
```

Comment:
ChainerRL generally uses Google style docstrings https://github.com/google/styleguide/blob/gh-pages/pyguide.md#381-docstrings

> A docstring should be organized as a summary line (one physical line) terminated by a period, question mark, or exclamation point, followed by a blank line, followed by the rest of the docstring starting at the same cursor position as the first quote of the first line.

I think it is not good to continue lines from the first line.

Reply:
Ok. Will change.